src/components/Datatable/Datatable.stories.tsx

```diff
@@ -181,18 +180,29 @@ export const Default: Story = () => {
     },
     [setIsLoading, setTableData],
   );
+  const logSelectedRows = useCallback((ids, exclude) => {
+    console.log(
+      `OnSelect called for ids: ${ids} to be ${
+        exclude ? 'excluded' : 'included'
+      }`,
+    );
+  }, []);
```

Comment:
Here useCallback shouldn't help without dependency

Reply:
Yep doesn't make much sense here, it doesn't hurt either in this case, will remove